.github/workflows/build.yml

```diff
@@ -0,0 +1,264 @@
+
+      - name: Upload rpm package
+        uses: actions/upload-artifact@v2
+        if: ${{ matrix.goos == 'linux' && matrix.component == 'cli' && matrix.goarch == 'amd64'}}
+        with:
+          name: ${{ env.RPM_PACKAGE }}
+          path: out/${{ env.RPM_PACKAGE }}
+
+      - name: Test debian package
+        if: ${{ matrix.goos == 'linux' && matrix.component == 'cli' && matrix.goarch == 'amd64'}}
+        uses: addnab/docker-run-action@v3
+        with:
+          image: ubuntu:latest
+          options: -v ${{ github.workspace }}:/work
+          run: |
+            apt update && apt install -y openssl
+            cd /work
+            apt install ./out/${{ env.DEB_PACKAGE }}
+            CONSUL_K8S_VERSION="$(consul-k8s version | awk '{print $2}')"
+            VERSION="${{ needs.get-product-version.outputs.product-version }}"
+            if [ "${VERSION}" != "${CONSUL_K8S_VERSION}" ]; then
+              echo "Test FAILED, expected: ${VERSION}, got: ${CONSUL_K8S_VERSION}"
+              exit 1
+            fi
+            echo "Test PASSED, expected: ${VERSION}, got: ${CONSUL_K8S_VERSION}"
+
+      - name: Upload debian packages 
+        uses: actions/upload-artifact@v2
+        if: ${{ matrix.goos == 'linux' && matrix.component == 'cli' && matrix.goarch == 'amd64'}}
+        with:
+          name: ${{ env.DEB_PACKAGE }}
+          path: out/${{ env.DEB_PACKAGE }}
+
+  build-docker-default:
+    name: Docker ${{ matrix.arch }} default release build
+    needs: [get-product-version, build]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ["arm", "arm64", "386", "amd64"]
+    env:
+      repo: ${{ github.event.repository.name }}
+      version: ${{ needs.get-product-version.outputs.product-version }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Docker Build (Action)
+        uses: hashicorp/actions-docker-build@v1.2.1
+        with:
+          smoke_test: |
+            TEST_VERSION="$(docker run "${IMAGE_NAME}" consul-k8s-control-plane version  | awk '{print $2}')"
+            if [ "${TEST_VERSION}" != "${version}" ]; then
+              echo "Test FAILED"
+              exit 1
+            fi
+            echo "Test PASSED"
+          version: ${{ env.version }}
+          target: release-default
+          arch: ${{ matrix.arch }}
+          pkg_name: consul-k8s-control-plane_${{ env.version }}
+          bin_name: consul-k8s-control-plane 
+          workdir: control-plane
+          tags: |
+            docker.io/hashicorp/${{ env.repo }}-control-plane:${{ env.version }}
+            986891699432.dkr.ecr.us-east-1.amazonaws.com/hashicorp/${{ env.repo }}-control-plane:${{ env.version }}
+          dev_tags: |
+            hashicorppreview/${{ env.repo }}-control-plane:${{ env.version }}-dev
+            docker.io/hashicorppreview/${{ env.repo }}-control-plane:${{ env.version }}-${{ github.sha }}
+
+  build-docker-alternate:
```

Comment:
let's comment this out for now so that we don't inadvertently push a ubi image that we haven't tested yet. wdyt?

Reply:
Sounds good